### ISSUE TITLE
imagenet dataset question

### ISSUE THREAD

**Issue by gutouyu** [ftramer/slalom#5]
I see in the `python/scripts/eval.py` we are using imagenet dataset. I'm not an export in Deep Learning with Images, So do I have to download the whole imagenet2012 dataset? or there is an alternative dataset which is smaller I can use, because the imagenet dataset is so large and I have some issues download and store them.
Thank u very much.

**Comment by ftramer** [comment]
Slalom has been tested on a subset of the ImageNet validation dataset. I can't guarantee it will work out of the box with other datasets and model architectures, although the general techniques we used should still apply.

The validation set is not too hard to get (it's 10000 images if I remember correctly). I think there are instructions in Tensorflow's Github repository on how to download and preprocess the images.

**Comment by gutouyu** [comment]
So so so glad to hear from you, I have been stuck in here for two days. So I only need to download the origin whole validation dataset( it's indeed much smaller than the train dataset). and put it in the right place as the `eval.py` will load it . Am I right?
I have some confusions as below:
1. Do I need to get a subset of the Image validation dataset manually by myself? If so, is there any guide? I just don't know how to do it. Or the code will do it for me?
2. If I run vgg16 eval, are we going to train the model from the ground? If not , we don't have the model yet, how do we eval it with validation dataset?

Thank you very much. I'm just trying to understand your awesome work here.

**Comment by gutouyu** [comment]
I have found the answers about my questions:
1. yes! we need to handle the imagenet data ourselves. refer to this link:
`https://github.com/tensorflow/models/blob/master/research/inception/inception/data/process_bounding_boxes.py`
first we need to download the validation and bbox dataset. Then we need to transform them to `TfRecord` format because `tf.contrib.slim` only support this format input file.
2. Yes! we just use well trained vgg16 and other nets. Of course, we only care the forward pass in predicting.

**Comment by ftramer** [comment]
Sorry I missed that you had reopened this issue. Glad you figured it out!

**Comment by gutouyu** [comment]
It's ok, never mind :)

**Comment by bronzeMe** [comment]
> I have found the answers about my questions:
> 
> 1. yes! we need to handle the imagenet data ourselves. refer to this link:
>    `https://github.com/tensorflow/models/blob/master/research/inception/inception/data/process_bounding_boxes.py`
>    first we need to download the validation and bbox dataset. Then we need to transform them to `TfRecord` format because `tf.contrib.slim` only support this format input file.
> 2. Yes! we just use well trained vgg16 and other nets. Of course, we only care the forward pass in predicting.
@gutouyu Hi, I am meeting the same issue, could you  give some tips about how to download the Image  validation set, and how to convert it ?
